packages/flash/package.json

```diff
@@ -19,7 +19,6 @@
   },
   "devDependencies": {
     "@prairielearn/tsconfig": "workspace:^",
-    "@types/express-session": "^1.18.0",
```

Comment:
Does this need a changelog? I'm ok with a no, just making sure this is considered.

Reply:
Added changelog in 7d8bcbc.